core/src/main/scala/chandu0101/scalajs/react/components/mixins/TimerMixin.scala

```diff
@@ -1,31 +0,0 @@
-package chandu0101.scalajs.react.components
```

Comment:
im proposing to just delete this, and use [TimerSupport.scala](https://github.com/japgolly/scalajs-react/blob/topic/v0.10/extra/src/main/scala/japgolly/scalajs/react/extra/TimerSupport.scala) instead